data/src/main/java/me/androidbox/data/remote/network/EndPointConstant.kt

```diff
@@ -0,0 +1,8 @@
+package me.androidbox.data.remote.network
+
+object EndPointConstant {
+    const val REGISTER = "register"
+    const val LOGIN = "login"
+    const val LOGOUT = "logout"
+    const val AUTHENTICATE =  "authenticate"
+}
```

Comment:
I personally don't like having them like this. I usually prefer having the string directly in the api call (in this case, AuthenticationService)

So just: @POST("/register") and so on

Reply:
@MKiperszmid @philipplackner 
I would normally prefer to have all the paths in all one place so I can see all the them in one place